app/controllers/concerns/sufia/files_controller_behavior.rb

```diff
@@ -59,6 +61,9 @@ def load_resource_from_solr
 
     # routed to /files/new
     def new
+      find_collections_with_edit_access
+      default_option = SolrDocument.new(id: -1, title_tesim: t("sufia.upload.collection.select_default_option"))
```

Comment:
find_collections_with_edit_access is from Hydra:Collections.  65-66 add a default option with id=-1 and title=Select collection...  Are you sure you want the default added in Hydra:Collections?  If so, you could pass a parameter to find_collections_with_edit_access asking for a default to be added and possibly passing in a default value and title.  If one isn't provided, Hydra:Collections could default to something.  There are other times find_collections_with_edit_access is called when adding 'Select collection...' would not be appropriate.